Lesson9/merve/styles.css

```diff
@@ -0,0 +1,172 @@
+* {
+    box-sizing: border-box;
+  }
+  body{
+  max-width: 100%;
```

Comment:
this should not be needed however you have images and <pre> elements that are too wise so it creates horizontal scroll

Reply:
couldn't understand this section